### PR TITLE
Hajohns/catalog templates

### PR DESCRIPTION
**Pull request by haileyajohnson**
- allow user-supplied custom template fragments
- restyle catalog and dataset access pages

### CODE REVIEW

**Comment by lesserwhirls** [comment]
@haileyajohnson - to test locally, try:

` ./gradlew.bat :it:freshInstallTest`

**Comment by lesserwhirls** [comment]
Can you do the following:

1) In `gradle\any\dependencies.gradle`, bump the version of chronicle map to `net.openhft:chronicle-map:3.15.1` (currently 3.14.5)

   https://github.com/Unidata/thredds/blob/5.0.0/gradle/any/dependencies.gradle#L208

2) Also in dependencies.gradle, remove lines 212-218

   https://github.com/Unidata/thredds/blob/5.0.0/gradle/any/dependencies.gradle#L212-L218

**Comment by haileyajohnson** [comment]
Cool, I can access the ncss pages now and fixed the stuff I broke in those templates (and rebased on the right branch and now CI seems to be passing again).

File: tds/src/main/webapp/WEB-INF/altContent/startup/threddsConfig.xml [hunk 1]
```diff
@@ -48,10 +48,13 @@
      * that are HTML catalog views. The CSS document given in the "standardCssUrl"
      * element is used in all other generated HTML pages.
 
-    <standardCssUrl>tds.css</standardCssUrl>
-    <catalogCssUrl>tdsCat.css</catalogCssUrl>
-    <datasetCssUrl>tdsDst.css</datasetCssUrl>
-    <openDapCssUrl>tdsDap.css</openDapCssUrl>
+    LOOK
```
Comment (lesserwhirls) [inline tds/src/main/webapp/WEB-INF/altContent/startup/threddsConfig.xml]: ?

Comment (haileyajohnson) [reply tds/src/main/webapp/WEB-INF/altContent/startup/threddsConfig.xml]: Meant to remove both of these...

Comment (lesserwhirls) [reply tds/src/main/webapp/WEB-INF/altContent/startup/threddsConfig.xml]: Cool :-)

File: tds/src/main/webapp/WEB-INF/templates/pointFragments.html [hunk 2]
```diff
@@ -16,6 +23,7 @@
           west:  /*[[${#numbers.formatDecimal(boundingBox?.lonMin, 0, 3)}]]*/ "-90.0000",
           east:  /*[[${#numbers.formatDecimal(boundingBox?.lonMax, 0, 3)}]]*/ "90.0000"
       };
+      };
```
Comment (cwardgar) [inline tds/src/main/webapp/WEB-INF/templates/pointFragments.html]: Extra closing parenthesis here, FYI.

**Comment by lesserwhirls** [comment]
Thanks @cwardgar!

**Pull request merged**